### ISSUE TITLE
I can't build wxFormBuilder under windows 7 (MSYS2) with non monolithic build.

### ISSUE THREAD

**Issue by tomay3000** [wxFormBuilder/wxFormBuilder#360]
Hello,
I have tried everything to build wxFormBuilder under windows 7 (MSYS2) with non monolithic build, to include wxMediaCtrl in the release build.

The build commands mentioned in the documentation are no longer working.
How to build it from source to include wxMediaCtrl (non-monolithic build) using my wxWidgets build version (not the msys2 / mingw one) located in "C:\wxWidgets-3.0.3".

Thank you for your understanding.

**Comment by jhasse** [comment]
> The build commands mentioned in the documentation are no longer working.

You mean that they are no longer working when you're using your own wxWidgets?

**Comment by tomay3000** [comment]
I managed to get it compile.

The problem is in the premake ".lua" files in the lines that defines the libraries to link against (**libs	= "std,stc,richtext,propgrid,aui,ribbon,media"**), they are defined for use only when the "--force-wx-config" option is defined with "create_build_files4.bat" command, BUT when I omit the "--force-wx-config" option (because I have compiled wxWidgets myself and have no wx-config), the script will generate a wrong wxWidgets library list to link to.

So I have manually edited the generated ".make" files to include all the missing libraries (**LIBS      += -lticpp -lwx_mswu_plugin-interface-3.0 -lbfd -liberty -lpsapi -limagehlp -lintl -lwxbase30u -lwxbase30u_net -lwxbase30u_xml -lwxexpat -lwxexpatd -lwxjpeg -lwxjpegd -lwxmsw30u_adv -lwxmsw30u_aui -lwxmsw30u_core -lwxmsw30u_gl -lwxmsw30u_html -lwxmsw30u_media -lwxmsw30u_propgrid -lwxmsw30u_ribbon -lwxmsw30u_richtext -lwxmsw30u_stc -lwxmsw30u_webview -lwxmsw30u_xrc -lwxpng -lwxpngd -lwxregexu -lwxscintilla -lwxscintillad -lwxtiff -lwxtiffd -lwxzlib -lwxzlibd**), and this is only a workaround.

And the final patch has to be applied in the ".lua" files to make wxWidgets library list to link to **GENERIC** in all **scenarios**.

And finally, wxMediaCtrl will be included in the final wxFormBuilder build as you see in the screenshot:
![untitled](https://user-images.githubusercontent.com/10688223/36387253-b09c39a0-1598-11e8-83ac-188a844675aa.png)
